drizzlepac/haputils/align_utils.py

```diff
@@ -1345,19 +1345,47 @@ def update_image_wcs_info(tweakwcs_output, headerlet_filenames=None, fit_label=N
             cr2_comment = RMS_DEC_COMMENT.replace('mas', 'deg')
             hdulist[sci_extn].header['CRDER1'] = (info['RMS_RA'].deg, cr1_comment) if info['RMS_RA'] is not None else -1.0
             hdulist[sci_extn].header['CRDER2'] = (info['RMS_DEC'].deg, cr2_comment) if info['RMS_DEC'] is not None else -1.0
-            hdulist[sci_extn].header['NMATCHES'] = len(info['ref_mag']) if info['ref_mag'] is not None else 0
-            hdulist[sci_extn].header['FITGEOM'] = info['fitgeom'] if info['fitgeom'] is not None else 'N/A'
+            nmatch_val = len(info['ref_mag']) if info['ref_mag'] is not None else 0
+            hdulist[sci_extn].header['NMATCHES'] = nmatch_val
+            fitgeom_val = info['fitgeom'] if info['fitgeom'] is not None else 'N/A'
+            hdulist[sci_extn].header['FITGEOM'] = fitgeom_val
+
+            # adds rel keywords if they don't exist
+            keys_to_add = [
+                ("RELGEOM", "fitgeom for relative fit"),
+                ("RELMATCH", "number of matches for relative fit"),
+                ("RELRMS_D", "RMS in DEC of relative WCS fit(mas)"),
+                ("RELRMS_R", "RMS in RA of relative WCS fit(mas)"),
+                ("RELREFIM", "base reference image rootname for relative fit"),
+            ]
+            for keyword in keys_to_add:
+                if keyword[0] not in hdulist[sci_extn].header:
+                    hdulist[sci_extn].header.insert(
+                        "FITGEOM", (keyword[0], "N/A", keyword[1]), after=True
+                    )
```

Comment:
The RELMATCH, RELRMS_D, and RELRMS_R keywords represent numeric values.   Even though these keywords may not be applicable in any particular processing, they should not contain the string value of "N/A".  REMATCH=0, RELRMS_D=RELRMS_R=0.0 will/should be obvious to the user that the zero values are placeholders or defaults as this is common practice.

Reply:
The "N/A" was specifically requested in the Jira ticket but I'm happy to change them to zeros if it's breaking a fits norm/standard.

Reply:
Members of INS do not necessarily know the FITS standard.  If the keywords are numeric, then the values should not have strings as the values.  As the software engineer, you need to enforce the FITS standard.

Reply:
understand, thanks